<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupProperties</class>
 <widget class="QWidget" name="GroupProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>70</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="labelIterations">
       <property name="text">
        <string>Iterations</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ExpressionLineEdit" name="iterations"/>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="inlineScope"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelInlineScope">
       <property name="text">
        <string>Inline Scope</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpressionLineEdit</class>
   <extends>QLineEdit</extends>
   <header>session/ExpressionLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
